includes/class-wc-payments.php

```diff
@@ -844,6 +844,17 @@ public static function update_plugin_version() {
 		update_option( 'woocommerce_woocommerce_payments_version', WCPAY_VERSION_NUMBER );
 	}
 
+	/**
+	 * Sets the customer service instance. This is needed only for tests.
+	 *
+	 * @param WC_Payments_Customer_Service $customer_service_class Instance of WC_Payments_Customer_Service.
+	 *
+	 * @return void
+	 */
+	public static function set_customer_service( WC_Payments_Customer_Service $customer_service_class ) {
```

Comment:
Nit: Maybe move this after `get_customer_service`? So it's more grouped like `set_database_cache` and `set_account_service`.

Reply:
done